modules/core/src/v2/coins/eos.ts

```diff
@@ -217,4 +224,31 @@ export class Eos extends BaseCoin {
       throw new Error(`address validation failure: ${addressDetails.address} vs ${rootAddressDetails.address}`);
     }
   }
+
+  /**
+   * Assemble keychain and half-sign prebuilt transaction
+   *
+   * @param params
+   * @param params.txPrebuild {Object} prebuild object returned by platform
+   * @param params.prv {String} user prv
+   */
+  signTransaction(params) {
```

Comment:
Please add an interface for the param type and return type (both currently any). Most of the types you added are not needed and are inferred for you automatically.